doc/pages/upgrade/020-upgrade_011_10.md

```diff
@@ -0,0 +1,139 @@
+
+Leveraging the templating capabilities of Helm, the generated templates can be customized according
+to your needs. For instance, if you want to refer to a particular Operator's tag all you have to do
+is to append `--set image.tag=<the-required-tag>` to the previous command. To check all the
+configurable values run `helm show values astarte/astarte-operator`.
+
+Now it's time to prepare the cluster to allow the new operator installation by means of the
+templates generated in the previous step.
+
+The following tasks will be performed:
++ replacement of the Astarte's and AstarteVoyagerIngress' CRDs with their updated versions,
++ installation of the Flow's CRD,
++ installation of the Operator's RBACs,
++ installation of the Operator's Webhooks,
++ annotation of the installed resources as to allow Helm to manipulate and take control of them.
+
+This [bash
+script](https://github.com/matt-mazzucato/astarte-kubernetes-operator/blob/add-hack-script/hack/upgrade-operator-011-10.sh)
+takes care of handling the aforementioned tasks. Download it and, assuming you saved it as
+`upgrade-operator-011-10.sh`, run the following:
+```bash
+bash path/to/your/upgrade-operator-011-10.sh \
+    -d $ASTARTE_OP_TEMPLATE_DIR \
+    -n $ASTARTE_OP_RELEASE_NAME \
+    -N $ASTARTE_OP_RELEASE_NAMESPACE
+```
+
+In a more concise way you can perform the same task simply executing:
+```bash
+ASTARTE_OP_UPGRADE_SCRIPT_URL=https://raw.githubusercontent.com/matt-mazzucato/astarte-kubernetes-operator/add-hack-script/hack/upgrade-operator-011-10.sh
+curl -fsSL $ASTARTE_OP_UPGRADE_SCRIPT_URL > /tmp/upgrade-operator-011-10.sh
+bash /tmp/upgrade-operator-011-10.sh \
+    -d $ASTARTE_OP_TEMPLATE_DIR \
+    -n $ASTARTE_OP_RELEASE_NAME \
+    -N $ASTARTE_OP_RELEASE_NAMESPACE
+```
+
+## Install Astarte Operator v1.0
+
+Now it's time to install the Astarte Operator v1.0.
+
+If during the [preparation of the cluster](#prepare-the-cluster-to-the-operator-v1-0-installation)
+you customized the chart values with the `--set` flag, please take care of setting the same values
+accordingly while installing Astarte Operator.
+
+To install the Operator, simply run:
+
+```bash
+helm install $ASTARTE_OP_RELEASE_NAME astarte/astarte-operator -n $ASTARTE_OP_RELEASE_NAMESPACE \
+    --version 1.0.0 --skip-crds
+```
+
+Note that the `--skip-crds` flag is required as, following the migration path, we already
+updated/installed the required CRDs.
+
+Version `1.0.0` is a safe landing version to perform the upgrade to. Upgrading to a more recent
+version may lead to a broken state with possible catastrophic aftermaths: if you choose to follow
+this path, make sure you know what you are doing. However deviating from the upgrade path outlined
+within the current page is strongly discouraged.
+
+After the successful migration to v1.0, please upgrade to a more recent Operator's version following
+the instructions outlined in the [Upgrade Operatori v1.0](./030-upgrade_10.html) section.
```

Comment:
Add a section detailing the naming changes of the services, so that users which deployed their own custom ingress are aware of them.